File: DiscordBot/src/main/java/com/example/demo/Listeners/implementations/PingListenerImpl.java
package com.example.demo.Listeners.implementations;

import com.example.demo.Listeners.PingListener;
import org.javacord.api.event.message.MessageCreateEvent;
import org.springframework.stereotype.Component;

@Component
public class PingListenerImpl implements PingListener {
    @Override
    public void onMessageCreate(MessageCreateEvent event) {
        if(event.getMessageContent().equalsIgnoreCase(";;ping")) {
            event.getChannel().sendMessage("PONG!");
        }
    }
}

File: KotlinBot/src/main/kotlin/com/example/KotlinBot/Listener/Implementation/PingListenerImpl.kt
package com.example.KotlinBot.Listener.Implementation

import com.example.KotlinBot.Listener.PingListener
import org.javacord.api.entity.channel.TextChannel
import org.javacord.api.event.message.MessageCreateEvent
import org.springframework.stereotype.Service

@Service
class PingListenerImpl : PingListener {
    override fun onMessageCreate(event: MessageCreateEvent?) {
        if (event != null) {
            if(event.messageContent == ";;ping") {
                event.channel.sendMessage("PONG!");
                event.channel.sendMessage("channel id = " + event.channel.id)
            }
        }
    }
}